src/drivers/better-sqlite3/index.ts

```diff
@@ -31,6 +31,8 @@ export const electrify = async (db: Database, config: ElectricConfig, opts?: Ele
   const namespace = new ElectricNamespace(adapter, notifier)
   const electric = new ElectricDatabase(db, namespace)
 
-  const electrified = await baseElectrify(dbName, db, electric, adapter, migrator, notifier, socketFactory, registry, config)
+  const configWithDefaults = addDefaultsToElectricConfig(config)
```

Comment:
Why are you adding the defaults only in `better-sqlite3` driver? Should other packages be treated differently for some reason? I'd personally pull this call into `baseElectrify` instead

Reply:
yeah, that was the intention but I forgot. Fixed now.